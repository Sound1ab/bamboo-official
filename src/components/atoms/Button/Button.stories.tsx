import { storiesOf } from '@storybook/react'
import * as React from 'react'
import { Button } from './Button'

storiesOf('Button', module)
  .add('Primary', () => {
    return (
      <React.Fragment>
        <Button variant="primary">Normal button</Button>
        <Button variant="primary">A button with way too long text</Button>
        <Button variant="primary">min width</Button>
      </React.Fragment>
    )
  })
  .add('Secondary', () => {
    return (
      <React.Fragment>
        <Button variant="secondary">Normal button</Button>
        <Button variant="secondary">A button with way too long text</Button>
        <Button variant="secondary">min width</Button>
      </React.Fragment>
    )
  })
  .add('Number', () => {
    return (
      <React.Fragment>
        <Button variant="number">1</Button>
        <Button variant="number">2</Button>
        <Button variant="number">3</Button>
      </React.Fragment>
    )
  })
