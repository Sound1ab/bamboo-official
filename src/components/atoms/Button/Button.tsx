import * as React from 'react'
// import styled from 'react-emotion'

interface PropTypes {}

export const Button: React.SFC<PropTypes> = () => <div>hey there you</div>
