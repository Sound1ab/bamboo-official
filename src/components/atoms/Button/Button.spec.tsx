import * as React from 'react'
import { render, cleanup } from '../../../test-utils'
import { Button } from './Button'

afterEach(cleanup)

describe('Button', () => {
  it('should match a snapshot', () => {
    const test = render(<Button />)
    expect(test.baseElement).toMatchSnapshot()
  })
})
