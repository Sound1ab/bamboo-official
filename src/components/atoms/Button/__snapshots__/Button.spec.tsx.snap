// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match a snapshot 1`] = `
<body>
  <div>
    <div>
      hey there you
    </div>
  </div>
</body>
`;
